numpy/tests/test_scripts.py

```diff
@@ -0,0 +1,79 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+""" Test scripts
+
+Test that we can run installed scripts
+"""
+from __future__ import division, print_function, absolute_import
+
+import os
+from os.path import join as pjoin, isfile, dirname, basename
+import sys
+from subprocess import Popen, PIPE
+
+import numpy as np
+from numpy.compat.py3k import basestring, asbytes
+
```

Comment:
I put the blank lines in to break up import blocks into stdlib, common third-party, and testing.  But happy to take them out if you prefer.